serverless-scheduler-proxy/main.go

```diff
@@ -88,45 +89,70 @@ func main() {
 
 func rewriteBotCronURL(c botConfig) func(*http.Request) {
 	return func(req *http.Request) {
-
-		var bodyBytes []byte
-		if req.Body != nil {
-			bodyBytes, _ = ioutil.ReadAll(req.Body)
-			req.Body = ioutil.NopCloser(bytes.NewBuffer(bodyBytes))
-		} else {
-			log.Println("request had no body")
+		req.Header.Add("x-github-event", "schedule.repository")
+		parser := func(bodyBytes []byte) (string, string) {
+			var pay reqPayload
+			json.Unmarshal(bodyBytes, &pay)
+			return pay.Name, pay.Location
 		}
+		rewriteBotURL(c, parser, req)
+	}
+}
 
-		var pay reqPayload
-		json.Unmarshal(bodyBytes, &pay)
+func rewriteBotPubSubURL(c botConfig) func(*http.Request) {
```

Comment:
we're going to be able to use this for both `pubsub` and `cron`? I noticed they share a helper.